signer/client/signer_trust.go

```diff
@@ -180,16 +179,7 @@ func (trust *NotarySigner) ListAllKeys() map[string]string {
 // CheckHealth checks the health of one of the clients, since both clients run
 // from the same GRPC server.
 func (trust *NotarySigner) CheckHealth(timeout time.Duration) error {
-
-	// Do not bother starting checking at all if the connection is broken.
-	if trust.clientConn.State() != grpc.Idle &&
-		trust.clientConn.State() != grpc.Ready {
-		return fmt.Errorf("Not currently connected to trust server.")
-	}
-
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	// the kmClient should point at the same server as the sClient, so a single
-	// healthcheck should suffice
```

Comment:
This one might make sense to leave because folks might assume `kmClient` and `sClient` should both have health checks, if they were different services.